src/overview/onboarding/actions.js

```diff
@@ -35,8 +35,37 @@ export const init = () => (dispatch, getState) => {
 
 export const toggleShouldTrack = () => async (dispatch, getState) => {
     const toggled = !selectors.shouldTrack(getState())
-    await persistShouldTrack(toggled)
-    dispatch(setShouldTrack(toggled))
+
+    const trackEvent = force => {
+        const trackEvent = analytics.trackEvent(
```

Comment:
@poltak We are using this function two times, here and in the privacy, should we make single and use at both places?
For now, it's ok, but there is a scope of improvement. :slightly_smiling_face:

Reply:
Yeah, it's pretty much the same thing, just different redux actions dispatched. Maybe it can just be a function that takes a boolean, and maybe an action to dispatch or just a callback? Not sure where we would put it though...

Reply:
For now, we can have just like this, but in #440 I've written the function, then we can use at both places. :slightly_smiling_face: